aten/src/ATen/native/Activation.cpp

```diff
@@ -424,6 +425,14 @@ Tensor hardswish_backward(const Tensor& grad_output, const Tensor& self) {
   return iter.output();
 }
 
+std::tuple<Tensor, Tensor> attn(const Tensor & q, const Tensor & k, const Tensor & v){
+  auto k_t = at::transpose(k, 0, 1);
```

Comment:
I would expect to see some TORCH_CHECK to make sure the size of the inputs and their number of dimensions is what you expect!